.github/workflows/test.yaml

```diff
@@ -4,6 +4,30 @@ on:
   workflow_call:
 
 jobs:
+  inclusive-naming-check:
+    name: Inclusive naming
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: canonical-web-and-design/Inclusive-naming
+      - run: mv * /tmp
+      - uses: actions/checkout@v3
+      - name: Merge configuration files
+        run: |
+          yq eval-all '
+          (
+            . as $item ireduce ({}; . *+ $item) | .rules | unique_by(.name)
+          ) as $mergedArray | . as $item ireduce ({}; . *+ $item) | .rules = $mergedArray
+          ' .woke.yaml /tmp/config.yml > /tmp/config.yaml
+          cat /tmp/config.yaml
```

Comment:
This is great in terms of being able to merge what's defined in that repo with what's in our location `.woke.yml`. I worry a little that this will mean that we can't run the check locally, and we'll get different results in CI than locally, but it's a minor concern. If there's a way to do this that could end up with the same result then great, but I'm not sure that's possible given this is using an externally defined github action.

Reply:
I agree. However, I'm not sure we'll be able to do so given that the list is different from the default snap configuration